specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2016-08-10/examples/ReplicationJobs_Export.json

```diff
@@ -30,8 +30,8 @@
           "targetObjectName": "",
           "targetInstanceType": "Other",
           "customDetails": {
-            "blobUri": "https://seapod01srs1pspk73202.blob.core.windows.net/datacontainer/Export_37e0fc2b-13f2-4817-aafa-0cd807d46842",
```

Comment:
Please change to place-holder

Reply:
@PhoenixHe-msft :Added place holder values